gp-includes/things/translation-set.php

```diff
@@ -137,12 +137,15 @@ public function import( $translations ) {
 		$locale = GP_Locales::by_slug( $this->locale );
 		$user = wp_get_current_user();
 
+		$existing_translations = array();
+
 		$current_translations_list = GP::$translation->for_translation( $this->project, $this, 'no-limit', array('status' => 'current', 'translated' => 'yes') );
-		$current_translations = new Translations();
+		$existing_translations['current'] = new Translations();
```

Comment:
None other than unifying all existing translations, grouped by status in a single array.